### DISCUSSION TITLE
Is there a way to give multiple values to a single modifier?

### DISCUSSION THREAD

**Discussion by Sleepful** [tailwindlabs/tailwindcss#9510] [Help]
Currently I have the following class:
```
[&_input]:bg-red-200

```

the first part is a descent selector for input fields, lets call it the "modifier", the second part is the value for the modifier. Maybe I want to also add:

```
[&_input]:border-2

```

Ok nice so this is what it looks like together:

```
class="[&_input]:bg-red-200 [&_input]:border-2"

```

now it is starting to get verbose, especially if i want to keep adding more properties to the same modifier. 

Is there a way to stack these together? Something like:

```
class="[&_input]:(bg-red-200,border-2)"
```

**Comment by wongjn** [comment]
[Here's a response to a similar suggestion](https://github.com/tailwindlabs/tailwindcss/discussions/9053#discussioncomment-3350470):

> We get this question so many times, so I would like to redirect you here for more information: https://twitter.com/adamwathan/status/1461519820411789314
